src/mito/Cargo.toml

```
@@ -33,8 +33,6 @@ snafu.workspace = true
 storage = { path = "../storage" }
 store-api = { path = "../store-api" }
 table = { path = "../table" }
-tempdir = { version = "0.3", optional = true }
+tempfile.workspace = true
+common-test-util = { path = "../common/test-util" }
 tokio.workspace = true
```

This only imports common-test-util to `[dependencies]` when we enable the `test` feature
```suggestion
common-test-util = { path = "../common/test-util", optional = true }
tokio.workspace = true

[dev-dependencies]
common-test-util = { path = "../common/test-util" }
```